.github/workflows/continous-integration.yml

```diff
@@ -1,8 +1,14 @@
 name: continuous_integration
 
 on:
+  workflow_dispatch:
+    inputs:
+      push-coverage:
+        description: "Boolean flag to turn on/off the coveralls.io step"
+        required: false
+        default: "false"
```

Comment:
Currently using `false` as a default because this fork doesn't have credentials for coveralls, and I'd like to be able to demo a passing CI.

This could be changed to `true` prior to merge, and coveralls would attempt to run every time the CI runs, unless it's a `workflow_dispatch` and the user opts not to run it.